modules/appeals_api/app/swagger/appeals_status/v1/swagger.json

```diff
@@ -48,7 +48,7 @@
           {
             "name": "icn",
             "description": "Veteran's Master Person Index (MPI) Integration Control Number (ICN)",
-            "example": "1234567890V543210",
+            "example": "1012832025V743496",
```

Comment:
This is the ICN that has a mocked response in the sandbox environment that returns 1 Higher-Level Review, 1 Supplemental Claim, and 2 Legacy Appeals.